File: user_input.cs
// using System;
class UserInput{
    static void Main(){
        Console.Write("Enter your name : ");
        string a = Console.ReadLine();
        Console.WriteLine("Hello " + a);

        Console.Write("Enter Age : ");
        int b = Convert.ToInt32(Console.ReadLine());
        Console.WriteLine("Age is : " + b);

        // float 

        Console.Write("Enter height : ");
        float c = float.Parse(Console.ReadLine());
        Console.WriteLine("Age height  : " + c);
    }
}

File: UserInput.java
import java.util.Scanner;;
public class UserInput {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter your name : ");
        String name = sc.nextLine();
        System.out.println("Name is " + name);
        
        System.out.print("Enter your age : ");
        int n = sc.nextInt();
        System.out.println("age is " + n);
        sc.close();
    }
}
